src/main/webapp/search-forms/editor.tsx

```diff
@@ -109,11 +110,16 @@ const searchFormToSearch = (form: QueryType) => {
   }
 }
 
-export const SearchFormEditor = (props: EditorProps) => {
-  const [searchForm, setSearchForm] = useState(props.form || {})
+export const QueryEditor = (props: EditorProps) => {
+  const query = props.query || {}
+  const queryBuilder = props.attributeDefinitions
+    ? React.cloneElement(props.queryBuilder, {
+        attributeDefinitions: props.attributeDefinitions,
+      })
+    : props.queryBuilder
```

Comment:
What is this for?

Reply:
I wasn't sure if it was better to add attributeDefinition props to the query builder here, or in route.tsx